ctapipe/tools/fit_spe.py

```diff
@@ -0,0 +1,219 @@
+"""
+Extract the Single Photoelectron spectrum parameters from the measured charges
+stored in a DL1 file
+"""
+import tables
+from tables.nodes import filenode
+from traitlets import Dict, List, Unicode, Int, Tuple
+from spefit import CameraFitter, PDF, Cost
```

Comment:
since this is now an optional dependency, it might be good to wrap this in a try/except ImportError block and emit a user-friendly message like "please install the spefit package (`conda install -c cta-observatory spefit` or `pip install spefit`".  The ctapipe conda package will not install the optional deps by default.